### ISSUE TITLE
Add in RTT analysis also 

### ISSUE THREAD

**Issue by 4a616d6573205265696c6c79** [4a616d6573205265696c6c79/tartiflette#2]
We're currently doing analysis of the hops only, we should also add in RTT analysis at the same time. This may involve fairly large changes to how the data is structured and thus the core logic.

**Comment by romain-fontugne** [comment]
Yes! But I think this is a different problem, so it is probably better to implement it in a different script. 

The detection of RTT change should be easy (using similar technique to hop only analysis) but after detecting a change we would like to find how the path changed, which seems a bit more challenging.
